### ISSUE TITLE
PRC Composition

### ISSUE THREAD

**Issue by lindsmith** [ga4gh/product-process#21]
Specific Notes 6ii details a new PRC composition:

1. A leader from one other Work Stream
2. A member of a third, different, Work Stream
3. A representative of the wider community of adopters, possibly from one of the Driver Projects requiring the product or another adopter

While requiring two work stream members (lead + other member) could help with xWS coordination, it also removes a slot for a potential adopter or expert external to GA4GH (which was encouraged in the old structure).  I wonder if the adopter/external community should have more representation because they can comment on usability, how well the standard fits needs, bring in a new perspective, etc. 

In other words, do we want to emphasize internal GA4GH standards interop, or value to the wider community? (Both are important, just throwing this out there as food for thought).

**Comment by susanfairley** [comment]
Thanks for these comments @lindsmith 

For your second point, where Work Stream Leads may not be available, I think opening things up to allow others (subgroup leads or someone nominated by the WSL perhaps) makes sense. The important thing, I think, is that it is someone with appropriate knowledge and understanding, including context in GA4GH and beyond, for the standard.

Your first point on GA4GH interoperability v community input seems trickier to me. Really, we want both.

Looking again at the process currently described, the references to interoperability focus on the implementations of the standard under discussion, not coordination between standards (within GA4GH and elsewhere), when at the standard review stage. There is, however, a requirement for the study group to speak to how the product would 'interoperate' within the pre-existing landscape when the study outputs are reviewed, so I think we are asking people to present evidence of the interaction with the wider landscape.

Maybe we need:

1. At least one person who has been heavily involved in GA4GH for more than a year
2. At least one person who does not have close involvement in GA4GH
3. The requirement for appropriate domain knowledge obviously remains for all
4. The requirement for a WSL is relaxed to allow others in GA4GH, possibly nominated by the WSLs of a related WS, to fill the role
5. We let domain knowledge, breadth of representation, etc determine if the third member is someone involved in GA4GH or not

Some rough ideas. Happy to hear opinions from others.

**Comment by heidirehm** [comment]
I agree we need to relax the requirement for another WS lead as we ask a lot of our WSLs and not overburdening them is also a factor.

**Comment by andrewyatz** [comment]
I'm not sure the overburden is the big issue, but a WSL feeling confident enough to comment directly on the product being reviewed. The WS have different sizes of overlap meaning some have to pick from a small pool of potential candidates. The WSL role was meant to ensure that product quality has been maintained i.e. the product/standard is of a level we expect from those passing PRC and to bring their GA4GH experience. 

But based on the rough ideas given above by @susanfairley I agree with all of them and would provide a good PRC process. We still need WSLs to proactively help to fill PRC roles.

**Comment by susanfairley** [comment]
Have attempted a redraft of the PRC composition criteria, aiming to reflect the above discussion. Link to the commit is above.
